jdaviz/configs/imviz/plugins/catalogs/catalogs.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import numpy.ma as ma
+
+from traitlets import List, Unicode, Bool, Int, observe
+
+from astropy.table import QTable
+from astropy.coordinates import SkyCoord
+from astroquery.sdss import SDSS
+
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import PluginTemplateMixin, ViewerSelectMixin
+
+__all__ = ['Catalogs']
+
+
+@tray_registry('imviz-catalogs', label="Imviz Catalogs")
+class Catalogs(PluginTemplateMixin, ViewerSelectMixin):
+    template_file = __file__, "catalogs.vue"
+    catalog_items = List([]).tag(sync=True)
+    catalog_selected = Unicode("").tag(sync=True)
+    results_available = Bool(False).tag(sync=True)
```

Comment:
is `results_available` used for anything that can't just rely on checking if `results_available > 0`?

Reply:
So, `results_available` is set to `True` when either the number of results is 0 or greater. I guess both variables are not necessary if I am able to adjust the .vue file to output a value to the user in the case of results `>= 0`.

Reply:
However, the initialization of `number_of_results` is set to 0, so even if no data is given (for example), then with this change, it'll still output the number of results being 0 when it's technically not the case.

Reply:
ok, as long as they serve separate purposes, then it's ok to leave them both in.  Thanks for the explanation!